api/src/Entity/Book.php

```diff
@@ -29,9 +29,10 @@ class Book
     /**
      * @var int
```

Comment:
Using integer ids is not a good practice in such API, and I think it might be disturbing if we let integer ids & UUIDs in the same repository.
As @dunglas said in the following comment, we should encourage using UUIDs.